### PR TITLE
Change StratifiedShuffleSplit to ttrain_test_split

### PR DESCRIPTION
**Pull request by pronojitsaha**
Addresses #99

### CODE REVIEW

**Comment by rhiever** [comment]
Hmmm... it's saying that no changes were made. Did you update from `master` and overwrite your changes?

**Comment by pronojitsaha** [comment]
In hurry, forgot to commit! Should be good now.

**Comment by rhiever** [comment]
Thanks! Did you verify that it returns the same splits? If not, we'll have to verify that before merging.

**Comment by rhiever** [comment]
Just checked and, by default, `train_test_split` doesn't stratify the data by class. You have to pass the `stratify` option and a list of the class labels, e.g.,

```python
X_train, X_test, y_train, y_test = train_test_split(input_data.drop('class', axis=1).values, 
                                                    input_data['class'].values,
                                                    train_size=0.75, test_size=0.25,
                                                    random_state=RANDOM_STATE,
                                                    stratify=input_data['class'].values)
```

**Comment by pronojitsaha** [comment]
Ok..sure.

**Comment by pronojitsaha** [comment]
I have checked the splits, and the split ratio is same for both `train_test_split` and `StratifiedShuffleSplit`, but the split indices are somewhat different which is expected.

**Comment by rhiever** [comment]
If you set the `random_state` to the same thing in your tests, they should come out the same. That's what I verified on my end yesterday.

File: tpot/tpot.py [hunk 1]
```diff
@@ -214,10 +214,10 @@ def fit(self, features, classes):
             np.random.shuffle(data_columns)
             training_testing_data = training_testing_data[data_columns]
 
-            training_indices, testing_indices = next(iter(StratifiedShuffleSplit(training_testing_data['class'].values,
-                                                                                 n_iter=1,
+            training_indices, testing_indices = train_test_split(training_testing_data.index,
```
Comment (rhiever) [inline tpot/tpot.py]: This doesn't look right. The call needs to look something like:

```python
(training_features, testing_features,
training_labels, testing_labels) = train_test_split(input_data.drop('class', axis=1).values, 
                                                    input_data['class'].values,
                                                    train_size=0.75, test_size=0.25,
                                                    random_state=RANDOM_STATE,
                                                    stratify=input_data['class'].values)
```

Have you tested this?

Comment (pronojitsaha) [inline tpot/tpot.py]: Yes, I have tested it on IRIS and MNIST dataset. Works the same. We can also do it the way you have pointed out, but using `training_testing_data.index` to get `training_indices, testing_indices` is in line with rest of our code format.

**Comment by pronojitsaha** [comment]
Ok, got that. Thanks.

**Comment by rhiever** [comment]
Understood. Alright, looks good to merge. Thanks again! :-)
